sdk/appconfiguration/feature-management/README.md

```diff
@@ -0,0 +1,218 @@
+
+#### Targeting Filter
+
+The Targeting Filter enables a feature flag based on a list of users, groups, or rollout percentages. It has a parameter:
+
+- `Audience` - The audience of the feature flag.
+
+Which contains the following fields:
+
+- `Users` - A list of users that the feature flag is enabled for.
+- `Groups` - A list of groups that the feature flag is enabled for, at different percentages.
+- `DefaultRolloutPercentage` - A percentage value that the feature flag is enabled for.
+- `Exclusions` - An object that contains a list of users and groups that the feature flag is disabled for.
+
+```json
+{
+    "name": "Microsoft.TargetingFilter",
+    "parameters": {
+        "Users": ["user1", "user2"],
+        "Groups": [
+            {
+                "Name": "group1",
+                "RolloutPercentage": 100
+            }
+        ],
+        "DefaultRolloutPercentage": 50,
+        "Exclusion": {
+            "Users": ["user3"],
+            "Groups": ["group2"]
+        }
+    }
+}
+```
+
+The `Users` field is a list of strings that represent the users that the feature flag is enabled for. The `Groups` field is a list of objects that represent the groups that the feature flag is enabled for. The `Name` field is a string that represents the name of the group. The `RolloutPercentage` field is a number between 0 and 100 that represents the percentage of users in the group that the feature flag is enabled for. The `DefaultRolloutPercentage` field is a number between 0 and 100 that represents the percentage of users that the feature flag is enabled for if they are not in any of the groups. The `Exclusions` field is an object that contains a list of users and groups that the feature flag is disabled for. The `Users` field is a list of strings that represent the users that the feature flag is disabled for. The `Groups` field is a list of strings that represent the groups that the feature flag is disabled for.
+
+#### Custom Filters
+
+You can also create your own feature filters by implementing the `FeatureFilter` interface.
+
+```python
+class CustomFilter(FeatureFilter):
+    def evaluate(self, context, **kwargs):
+        ...
+        return True
+```
+
+The `evaluate` method is called when checking if a feature flag is enabled. The `context` parameter contains information about the feature filter from the `parameters` field of the feature filter. Any additional parameters can be passed in as keyword arguments when calling `is_enabled`.
+
+You can then pass your custom filter into the `FeatureManager` when it is initialized.
+
+```json
+{
+    "name": "MyCustomFilter",
+    "parameters": {
+        ...
+    }
+}
+```
+
+`parameters` takes any additional parameters that are needed for the feature filter as a dictionary. These parameters are passed into the `evaluate` method.
+
```

Comment:
We need some documentation for requirement_type

Reply:
It is kept like this as Python is like DotNet and has issues loading the variable from the service. So, I have the code for it now, but we will not fully expose it till it can be use with the service.